### DISCUSSION TITLE
Getting test results from failed CI tests

### DISCUSSION THREAD

**Discussion by tonygermano** [mozilla/rhino#1602] [Q&A] [answered]
I can open an issue if I need to, but I wanted to get some feedback first. When I first opened https://github.com/mozilla/rhino/pull/1594, the automated build failed. The run results are at https://github.com/mozilla/rhino/actions/runs/10622780835/job/29447815131. Near the end of the log after it failed, it says:
> There were failing tests. See the report at: file:///home/runner/work/rhino/rhino/tests/build/reports/tests/test/index.html

Is there some way that I'm missing to view that report or does the github action need to be adjusted to allow those test results to be viewable?

**Comment by p-bakker** [comment]
If you go to the related GitHub Action, you can see the artifacts that are stored for each job (near the bottom) 

Not sure if the specific file you're referencing is already  part of the artifact though. If it's not, then you cannot get to it, but we should then modify the pipeline config to add it to the stored artifacts I think

**Reply by tonygermano** [reply]
Yeah, I think something needs to be adjusted. I see now on a successful test I see where to get the artifact download link as part of the Upload Results step, but when the tests failed, that step didn't run. I'm going to convert this discussion to an issue.

**Comment by tonygermano** [comment, marked as answer]
This has been moved to #1606
